src/qrcode/QRScanner.tsx

```diff
@@ -59,6 +59,11 @@ export default function QRScanner({ onBarCodeDetected }: QRScannerProps) {
   const [value, setValue] = useState('')
   const [displayEntryModal, setDisplayEntryModal] = useState(false)
 
+  // Handles explorer QR Codes & same behavior as scanner another users QRCode
```

Comment:
is there any business requirement to support QR codes from explorer? i think it's a nice initiative but explorer gives a QR code for any address, which includes tokens for example https://alfajores-blockscout.celo-testnet.org/token/0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F/token-transfers - does it make sense for us to redirect the user to the payment screen if they accidentally scan an address like this?

Reply:
Removed in https://github.com/valora-inc/wallet/pull/2529/commits/c2e6cec84fcf196d2ef7843ce8d5efa17a3b5bb5!

Reply:
chiming in super late here, but I don't know of any such requirement. in theory it would be nice to be able to support all QR codes from different sources such as explorer, but in practice it does not seem like something we urgently need to do right now.